index.js

```diff
@@ -5,14 +5,10 @@ var ect = require('ect');
 exports.name = 'ect';
 exports.outputFormat = 'html';
 exports.compile = function(str, options) {
-  if (!options) {
-    options = {};
-  }
-  if (!options.root) {
-    options.root = {
-      'page': str
-    };
-  }
+  options = options || {};
```

Comment:
@RobLoach @TimothyGu Wouldn't we be happy if we had something like `options ||= {}` or maybe even `options?=` ?sigh...